network/core-node/admin-guide/prerequisites.mdx

```diff
@@ -3,32 +3,79 @@ title: Prerequisites
+
+:::
+
+### Outbound
+
+- Stellar Core requires access to a database (PostgreSQL for example). If that database resides on a different machine on your network, you'll need to allow that connection. You'll specify the database when you [configure] Stellar Core.
+- You can safely block all other connections.
+
+## Storage
+
+Stellar Core stores two copies of the ledger: one in a SQL [database](#database) and one in XDR files on local disk called [buckets](#buckets). For the most part, the contents of both the database and buckets directories can be ignored, since they are completely managed by Stellar Core. In terms of storage space, 100 GB is enough in 2024. Most people seem to have at least 1TB on hand.
+
+### Database
+
+The database is consulted during consensus, and modified atomically when a transaction set is applied to the ledger. It's random access, fine-grained, and fast.
+
+While a SQLite database works with Stellar Core, we generally recommend using a separate PostgreSQL server. A Postgres database is the bread and butter of Stellar Core.
+
+If you're using PostgreSQL, we recommend you configure your local database to be accessed over a Unix domain socket, as well as updating the below PostgreSQL configuration parameters:
+
+```text
+# !!! DB connection should be over a Unix domain socket !!!
+# shared_buffers = 25% of available system ram
+# effective_cache_size = 50% of available system ram
+# max_wal_size = 5GB
+# max_connections = 150
+```
+
+### Buckets
+
+In addition to the SQL database, Stellar Core stores a duplicate copy of the ledger in the form of flat XDR files called "buckets." The bucket files are used for hashing and transmission of ledger differences to history archives.
 
-- **Outbound**:
-  - Stellar Core requires access to a PostgreSQL database. If that database resides on a different machine on your network, you'll need to allow that connection. You specify the database when you configure Stellar Core.
-  - You can block all other connections.
-- **Inbound**: Stellar Core exposes an _unauthenticated_ HTTP endpoint on its `HTTP_PORT`. You can specify a port when you [configure Stellar Core](./configuring.mdx), but most people use the default, which is **11626**.
-  - The `HTTP_PORT` is used by Horizon to submit transactions, so may have to be exposed to the rest of your internal IPs
-  - It's also used to query Stellar Core [info](./commands.mdx) and provide [metrics](./monitoring.mdx)
-  - And to perform administrative commands such as [scheduling upgrades](./network-upgrades.mdx) and changing log levels
-  - For more on that, see [commands](./commands.mdx)
+Buckets should be stored on a fast, local disk with sufficient space to store several times the size of the current ledger. Current ledger size is approximately 13 GB (as of 2024), so please plan accordingly.
```

Comment:
So ledger data 100GB in SQL and 13GB in buckets?  That may be the case, but I want to make sure that disparity is expected.

Reply:
I believe this is correct, but would love some confirmation from someone who knows better than me

Reply:
Ditto.